user-impl/src/bundle/bundle.conf

```diff
@@ -0,0 +1,28 @@
+version = "1"
+name = "user"
+compatibilityVersion = "1"
+system = "user"
+systemVersion = "1"
+nrOfCpus = 0.1
+memory = 268435456
+diskSpace = 200000000
+roles = ["web"]
+components = {
+  user = {
+    description = "user"
+    file-system-type = "universal"
+    start-command = ["user/bin/user"]
+    endpoints = {
+      "user" = {
+        bind-protocol = "http"
+        bind-port = 0
+        services = ["http://:9000/user","http://:9000/api/user?preservePath"]
```

Comment:
I think `"http://:9000/api/user?preservePath"` should not be listed since `UserSevice` has ACLs disabled.